### DISCUSSION TITLE
Start out with EJBCA Docker container

### DISCUSSION THREAD

**Discussion by hzami** [Keyfactor/ejbca-ce#582] [Q&A] [answered]
I am about starting EHBCA using Docker following this guide " https://doc.primekey.com/ejbca/tutorials-and-guides/tutorial-start-out-with-ejbca-docker-container " , In the 3dr step when I ran the command docker compose up -d, I got 
![Capture d'écran 2024-05-02 125458](https://github.com/Keyfactor/ejbca-ce/assets/53655669/963a77ef-82de-440e-90bb-076bb6597c8e)
Then I got 
![image](https://github.com/Keyfactor/ejbca-ce/assets/53655669/86f1d9fb-b452-46a7-9b83-6c7c37c6f741)
How can I find the right version that I should put

**Comment by primetomas** [comment, marked as answer]
See here, it seems as you can just remove the version line in the docker-compose.yml file.
https://github.com/mailcow/mailcow-dockerized/issues/5797

Try that and let us know it that works (so we can update the doc page).

**Reply by primetomas** [reply]
I've never seen that one. Looks like it's in the mariadb container. Might be good to create a new topic for this? To let this thread  be a good reference for the version line in docker-compose.

**Reply by svenska-primekey** [reply]
Yes please start a new topic for this new error.
